### PR TITLE
Routes on the meal sharing database for Node.js week-1

### PR DESCRIPTION
**Pull request by Nusrath-Jahan**
### **Routes**
/future-meals | Respond with all meals in the future (relative to the when datetime)
/past-meals | Respond with all meals in the past (relative to the when datetime)
/all-meals | Respond with all meals sorted by ID
/first-meal | Respond with the first meal (meaning with the minimum id)
/last-meal | Respond with the last meal (meaning with the maximum id)

### CODE REVIEW

File: api/.env-example [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 DB_HOST=localhost
 DB_PORT=3306
 DB_USER=root
-DB_PASSWORD=mysql-root-password
-DB_DATABASE_NAME=meal-sharing
+DB_PASSWORD=my-secret-pw
+DB_DATABASE_NAME=meal_sharing_website
 
 # The database you use for local development most likely doesn't use SSL (encryption)
 # but the one you deploy on Render.com does.
```
Comment (Rassibassi) [inline api/.env-example]: This file was supposed to be renamed from `.env-example` to `.env`. then `.env` is listed in your `.gitignore` here:
https://github.com/Nusrath-Jahan/meal-sharing/blob/main/api/.gitignore

This means your code repository never commits your passwords into the version control software `git`. Isn't that clever.

Here it seems you have shared your passwords with the world

File: api/src/database_client.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const connection = knex({
     host: process.env.DB_HOST,
     port: process.env.DB_PORT,
     user: process.env.DB_USER,
-    password: process.env.DB_PASSWORD,
+    password: process.Nj060099,
```
Comment (Rassibassi) [inline api/src/database_client.js]: This here is odd, if you have the password inside a `.env` file (not `.env-example`, remove the `example` from the name), then `process.env.DB_PASSWORD` should be able to pick up on the `DB_PASSWORD=...` line

**Review by Rassibassi — COMMENTED** [review]
hej!

your homework is copied 1 to 1 from here:
https://github.com/l0rians/meal-sharing/pull/1/files

File: api/.env-example (hunk 1 above)
Comment (Nusrath-Jahan) [reply api/.env-example]: Hello Rasmus,
At First I apologize for my homework that I copied from someone. 😞😞😞
Now I did it myself. I am really sorry for that.
I already have the .evn file that I attached to the .gitignore file. That's why I deleted the .env-example file.
And my database password, I don't use passwords anymore.
Thank you for your valuable suggestions.

**Pull request merged**